app/layout/scenarios/edit/analysis/adjust-planning-units/buttons/clicking/component.tsx

```diff
@@ -122,6 +120,9 @@ export const AnalysisAdjustClicking: React.FC<AnalysisAdjustClickingProps> = ({
           level: 'error',
         });
       },
+      onSettled: () => {
```

Comment:
About these... Not nitpicking, but when debugging I went on a different direction. Figured I might as well leave these changes in. I can removed them if needed; moved them into a separate commit which can be easily reverted.